lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/ExperianService.java

```diff
@@ -37,10 +37,13 @@ public String getResponseFromKBVExperianAPI(String payload, String uriEndpoint)
 
         HttpResponse<String> res =
                 this.httpClient.send(httpReq, HttpResponse.BodyHandlers.ofString());
-
-        LOGGER.debug(String.format("KBV Experian API response status code: %s", res.statusCode()));
         String body = res.body();
-        LOGGER.debug(String.format("KBV Experian API response: %s", body));
+
+        if (LOGGER.isDebugEnabled())
```

Comment:
If we're using PowerTools, then LOGGER.debug should already do this

Reply:
Added `EventProbe` as done in Address CRI